cypress.config.js

```diff
@@ -10,9 +10,16 @@ module.exports = defineConfig({
           const email = faker.internet.email();
           const randomNumber = Math.floor(Math.random(1000) * 1000);
```

Comment:
The Math.random() function returns a floating-point, pseudo-random number in the range 0 to less than 1. Therefore, Math.random(1000) will still generate a number from 0 to 1, not up to 1000. If you want to generate a number up to 1000, you should multiply the result of Math.random() by 1000, like this: `Math.floor(Math.random() * 1000)`. But in your case, you already do that, so the argument inside the Math.random function is redundant and can be removed.